ace.d.ts

```diff
@@ -883,10 +883,10 @@ export namespace Ace {
     jumpToMatching(select: boolean, expand: boolean): void;
     gotoLine(lineNumber: number, column: number, animate: boolean): void;
     navigateTo(row: number, column: number): void;
-    navigateUp(): void;
-    navigateDown(): void;
-    navigateLeft(): void;
-    navigateRight(): void;
+    navigateUp(times: number): void;
```

Comment:
`times` parameter is optional for all 4 methods, could you please update it to `times?: number`?

Reply:
Just updated. Thanks!